notebooks/localai/localai.ipynb

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
<p>can you please add some comments about used field in configuration file?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2564/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>